src/estimators/independent_set_estimator.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Estimator for sketches when using the independence assumption.
+
+Under the independence assumption, the cardinality of the union of two
+sketches representing sets A and B is equal to
+   
+  |A| + |B| - |A| * |B| / N,
+
+where N is the size of the universe from which A and B are drawn.
```

Comment:
@matthewclegg, @shengma-1 can you share any reference literature for this estimator? The logic makes sense to me and it seems fine as a baseline for our evaluation, but I am just not familiar with the theory behind this, so would love to learn more.

Reply:
The independence model assumes that ids in setA and ids in setB are independence draw from a universe (size=N).
Pr(ids in A and B) = Pr(ids in A) + Pr(ids in B) - Pr(Ids in A) * Pr(ids in B).
This paper: https://pdfs.semanticscholar.org/b0b1/34bf4f5d59c2c4f6e67b61f64dc1a1474646.pdf?_ga=2.21486968.293676905.1597883007-230818821.1551825210 shows that cross media reach is well estimated by the independence model.

Reply:
Sorry, the equation was wrong. It should be:
Pr(ids in A and B) = Pr(Ids in A) * Pr(ids in B).
Pr(ids in A or B) = Pr(ids in A) + Pr(ids in B) - Pr(Ids in A) * Pr(ids in B).

Reply:
Thank you Sheng! The paper is really clear.